src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -284,7 +306,27 @@ def prepopulate_cache(self, commit_sha):
         except Exception as e:
             logger.warn('Error while requesting coverage data', error=str(e))
 
-    def generate_zero_coverage_report(self, report):
+    def generate_per_suite_reports(self):
+        def generate_suite_report(suite):
+            output = self.generate_info(suite=suite, out_format='lcov')
```

Comment:
Well, *some* change here, since you removed the first `self.revision` parameter in this `self.generate_info` call.

Reply:
You're right, I have removed it because I made it optional (and it isn't used if `out_format` is `lcov' like in this case).